deps/amqp_client/src/amqp_connection.erl

```diff
@@ -151,8 +155,9 @@ start(AmqpParams) ->
 %% normal or an SSL connection.
 %% If ConnectionName is binary - it will be added to client_properties as
 %% user specified connection name.
-start(AmqpParams, ConnName) when ConnName == undefined; is_binary(ConnName) ->
+start(AmqpParams_, ConnName) when ConnName == undefined; is_binary(ConnName) ->
```

Comment:
This is a highly unusual choice of binding name. Consider using something like `AmqpParams0`.

Reply:
Will find another name: `AmqpParams0` is used a case, with the same `start/2`.